### PR TITLE
Box Retexture

### PR DESCRIPTION
**Pull request by CilliePaint**
# Description

Retextures Boxes slightly.

Description.

---

<details><summary><h1>Media</h1></summary>
<p>

Science
Standard
Olive
Doom

</p>
</details>

---

# Changelog

:CL:
- tweak: New Box Textures!

### CODE REVIEW

**Review by VMSolidus — APPROVED** [review]
Hilarious. In space, boxes have rounded corners for EFFICIENCY.

**Comment by FoxxoTrystan** [comment]
The Liscense will need to be changed by the orignal and who edited them.

Or if compleatly remade, who made them.

**Review by VMSolidus — DISMISSED** [review]
Please update the meta.json file in boxes.rsi to make sure that it says who edited the sprites from the original.

**Review by DangerRevolution — CHANGES_REQUESTED** [review]
You need to indicate where these new boxes are from

A) If you drew them, and you need to change the credit inside the meta.json file of each .rsi folder impacted to credit you for drawing them

B) You got them from someone or somewhere else, you need to change the credit inside the meta.json file of each .rsi folder impacted to credit whoever or wherever you got them from.

**Comment by CilliePaint** [comment]
> You need to indicate where these new boxes are from
> 
> A) If you drew them, and you need to change the credit inside the meta.json file of each .rsi folder impacted to credit you for drawing them
> 
> B) You got them from someone or somewhere else, you need to change the credit inside the meta.json file of each .rsi folder impacted to credit whoever or wherever you got them from.

shit oops, i'll get that fixed up

**Review by DangerRevolution — APPROVED** [review]

File: Resources/Textures/Objects/Storage/boxes.rsi/meta.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/cc65477c04f7403ca8a457bd5bae69a01abadbf0, encryptokey was taken from Baystation12 at https://github.com/infinitystation/Baystation12/blob/073f678cdce92edb8fcd55f9ffc9f0523bf31506/icons/obj/radio.dmi and modified by lapatison. boxwidetoy, shelltoy, swab, flare, inflatable, trashbag, magazine, holo and forensic created by potato1234x (github) for ss14 based on toys.rsi, mouth_swab.rsi, flare.rsi, inflatable_wall.rsi, trashbag.rsi, caseless_pistol_mag.rsi, guardians.rsi and bureaucracy.rsi respectively, candle and darts created by TheShuEd for ss14, throwing_knives and vials was drawn by Ubaser, evidence_markers by moomoobeef, Boxes by mefinks.",
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/Objects/Storage/boxes.rsi/meta.json:4]: Oh my god this is such a long line

**Review by DEATHB4DEFEAT — APPROVED** [review]

File: Resources/Textures/Objects/Storage/boxes.rsi/meta.json (hunk 1 above)
Comment (CilliePaint) [reply Resources/Textures/Objects/Storage/boxes.rsi/meta.json:4]: yeah it was nightmarishly long, one of these days i gotta retexture them all just so i can make it a little less text-wall

**Pull request merged**